bitrise.yml

```diff
@@ -858,27 +858,12 @@ workflows:
         run_if: '{{getenv "SSH_RSA_PRIVATE_KEY" | ne ""}}'
     - git-clone@6.2: {}
     - certificate-and-profile-installer@1: {}
-    - script@1:
+    - restore-cache@2:
         inputs:
-        - content: >-
-            #!/usr/bin/env bash
-
-            # fail if any commands fails
-
-            set -e
-
-            # debug log
-
-            set -x
-
-            curl --location --retry 5 --output l10n-screenshots-dd.zip
-            "$MOZ_DERIVED_DATA_PATH"
-
-            mkdir l10n-screenshots-dd
-
-            unzip l10n-screenshots-dd.zip -d l10n-screenshots-dd
-
-            rm l10n-screenshots-dd.zip
+        - key: |-
+            l10n-screenshots-dd-$GIT_CLONE_COMMIT_HASH
+            l10n-screenshots-dd-
```

Comment:
Just in case L10nScreenshots workflow is started after some changes is made, `restore-cache` should fall back to a cache with the prefix `l10n-screenshots-dd-`.

Reply:
If the `$GIT_CLONE_COMMIT_HASH` isn't defined, the fallback is `l10n-screenshots-dd-`. 🤷🏼‍♀️